e2e/tests/todo-columns.spec.ts

```
@@ -1,7 +1,28 @@
+import { createHash } from "crypto";
 import { test, expect, type Page } from "@playwright/test";
 import { mockAllApis } from "../fixtures/api";
 import { TODO_ITEMS, TODO_COLUMNS, type TodoFixture } from "../fixtures/todos";
```

ESLint

[error] Remove this unused import of 'TodoFixture'.

(sonarjs/unused-import)